pkg/api/v1alpha1/thirdparty/tinkerbell/zz_generated.deepcopy.go

```diff
@@ -19,8 +19,6 @@
 
 package tinkerbell
 
-import ()
```

Comment:
was this due to a kubebuilder change?

Reply:
this was due to `gci` tool, that fixes imports.

Reply:
Ah yeah. Could we revert it? We shouldn't change autogenerated files. The linter is already configured to omit these files.